Changelog.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+**1.4.2**
```

Comment:
Well actually, what do you mean? Don't you version accordingly to the upstreams versioning?